packages/node/src/methods/state-channel/create/controller.ts

```diff
@@ -56,6 +56,11 @@ export default class CreateChannelController extends NodeController {
       MinimumViableMultisig.abi
     ).functions.setup.encode([multisigOwners]);
 
+    let bytecode = Proxy.bytecode;
+    if (bytecode.slice(0, 2) !== "0x") {
+      bytecode = `0x${bytecode}`;
+    }
```

Comment:
This seems like a type failure. Why would it sometimes include `0x` and sometimes not?